contracts/apps/disputable/DisputableAragonApp.sol

```diff
@@ -0,0 +1,178 @@
+/*
+ * SPDX-License-Identitifer:    MIT
+ */
+
+pragma solidity 0.4.24;
+
+import "./IAgreement.sol";
+import "./IDisputable.sol";
+import "../AragonApp.sol";
+import "../../lib/token/ERC20.sol";
+import "../../lib/math/SafeMath64.sol";
+
+
+contract DisputableAragonApp is IDisputable, AragonApp {
+    /* Validation errors */
+    string internal constant ERROR_AGREEMENT_NOT_SET = "DISPUTABLE_AGREEMENT_NOT_SET";
+    string internal constant ERROR_AGREEMENT_ALREADY_SET = "DISPUTABLE_AGREEMENT_ALREADY_SET";
```

Comment:
In the interest of saving bytecode space, I wonder if we can combine the two errors above into just a single "AGREEMENT_STATE_INVALID" error or etc.